src/seedu/addressbook/data/AddressBook.java

```diff
@@ -24,13 +26,15 @@
 
     private final UniquePersonList allPersons;
     private final UniqueTagList allTags; // can contain tags not attached to any person
+    public ArrayList<Tagging> tagLog;
```

Comment:
Good, this is a correct place to put the list based on the UML.. May be tagList is a better name.